docs/releasing/publishing-a-pre-release.md

```diff
@@ -1,19 +1,31 @@
-# Publishing pre-release of GOV.UK Frontend
+# Publish pre-release of GOV.UK Frontend
 
-We use pre-releases when:
-- We are working on a new contribution and want to see it in the Design System website
-- We want to try an experimental feature as if it's pushed to npm.
+Before you use a pre-release, you need to make changes to GOV.UK Frontend. Then follow these instructions.
```

Comment:
@hannalaakso @36degrees Is there any content we could link to from the text "make changes to GOV.UK Frontend"?

Reply:
@EoinShaughnessy Maybe something like:
> Before you publish a pre-release, you will need to have [committed](https://github.com/alphagov/design-system-team-docs/blob/master/development/commit-messages.md) a code change to GOV.UK Frontend.

But the resource I linked to but might not be the best one as it's about writing a commit message rather than the mechanics of committing a change in git - we don't have guidance for the latter as it's beyond our team docs and depends what tool you're using for git etc. I think "committing a code change" could be enough in this instance.

Reply:
I would use the word "publish" instead of "use" here as it's more accurate.

Reply:
@hannalaakso Great suggestion! Makes sense to me. Will run by reviewer.